TestProjects/Playground/Playground.Droid/Views/DictionaryBindingView.cs

```diff
@@ -0,0 +1,58 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+using Android.App;
+using Android.Content;
+using Android.OS;
+using Android.Runtime;
+using Android.Util;
+using Android.Views;
+using Android.Widget;
+using Playground.Core.ViewModels;
+using MvvmCross.Droid.Views.Attributes;
+using MvvmCross.Droid.Support.V4;
+using MvvmCross.Binding.Droid.BindingContext;
+using MvvmCross.Binding.BindingContext;
+using Android.Graphics.Drawables;
+using Android.Graphics;
+
+namespace Playground.Droid.Views
+{
+    [MvxFragmentPresentation(typeof(RootViewModel), Resource.Id.content_frame, true,
+                         Resource.Animation.abc_fade_in,
+                         Resource.Animation.abc_fade_out,
+                         Resource.Animation.abc_fade_in,
+                         Resource.Animation.abc_fade_out)]
+    [MvxFragmentPresentation(typeof(SplitRootViewModel), Resource.Id.split_content_frame)]
+    [MvxFragmentPresentation(typeof(TabsRootViewModel), Resource.Id.content_frame)]
+    [MvxFragmentPresentation(fragmentHostViewType: typeof(ModalNavView), fragmentContentId: Resource.Id.dialog_content_frame)]
```

Comment:
Can you please cleanup these presentation attributes? There should be only one here